interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,184 @@
+                        <properties>
+                          <help>Zone caching options</help>
+                        </properties>
+                        <children>
+                          <leafNode name="on-reload">
+                            <properties>
+                              <help>Retrieval zone only at startup and on reload</help>
+                              <valueless/>
+                            </properties>
+                          </leafNode>
+                          <leafNode name="interval">
+                            <properties>
+                              <help>Periodic zone retrieval interval</help>
+                              <valueHelp>
+                                <format>&lt;minutes&gt;</format>
+                                <description>Retrieval interval in minutes</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;minutes&gt;m</format>
+                                <description>Retrieval interval in minutes</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;hours&gt;h</format>
+                                <description>Retrieval interval in hours</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;days&gt;d</format>
+                                <description>Retrieval interval in days</description>
+                              </valueHelp>
+                              <constraint>
+                                <regex>[1-9]([0-9]*)([mhd]{0,1})</regex>
+                              </constraint>
+                            </properties>
+                            <defaultValue>24h</defaultValue>
+                          </leafNode>
+                        </children>
+                      </node>
+                      <leafNode name="retry-interval">
+                        <properties>
+                          <help>Retry interval after zone retrieval errors</help>
+                          <valueHelp>
+                            <format>u32:1-2147483647</format>
+                            <description>Retry period in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-2147483647"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>60</defaultValue>
+                      </leafNode>
+                      <leafNode name="max-zone-size">
+                        <properties>
+                          <help>Maximum zone size in megabytes</help>
+                          <valueHelp>
+                            <format>u32:0-1024</format>
```

Comment:
I assume 0 refers to "ignore the size". If this is the case, please add a dedicated value help explaining the special meaning of 0

Reply:
fixed

Reply:
Plase make it even more obvious:

```xml
<valueHelp>
  <format>u32:0</format>
  <description>No restriction</description>
</valueHelp>
<valueHelp>
  <format>u32:1-1024</format>
  <description>Size in megabytes</description>
</valueHelp>
```

Reply:
fixed